app/NavComp.js

```diff
@@ -0,0 +1,80 @@
+'use client';
+
+import React, { useState } from 'react';
+import {
+    BookOutlined,
+  DashboardOutlined,
+  MenuFoldOutlined,
+  MenuUnfoldOutlined,
+  UploadOutlined,
+  UserOutlined,
+  VideoCameraOutlined,
+} from '@ant-design/icons';
+import { Button, Layout, Menu, theme } from 'antd';
+import Image from 'next/image';
+import { Footer } from 'antd/es/layout/layout';
+import Link from 'next/link';
+
+const { Header, Sider, Content } = Layout;
+
+const NavbarComp =
+ (props) => {
+  const [collapsed, setCollapsed] = useState(false);
+  const {
+    token: { colorBgContainer, borderRadiusLG },
+  } = theme.useToken();
+
+  return (
+    <Layout>
+      <Sider trigger={null} collapsible collapsed={collapsed}>
+        <div className="demo-logo-vertical my-6 mx-6">
+            <svg width="129" height="22" viewBox="0 0 129 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If svg files are heavy, its nice practice to add them to assets and fetch them here.